graph/GuidelinesGraph.md

```diff
@@ -326,6 +316,136 @@ For a complete mapping of error codes to HTTP statuses, see
 
 <a name="api-contract-and-non-backward-compatible-changes"></a>
 
+### Enums
```

Comment:
@OlgaPodo does this belong in the guidelines or as a pattern?  Works for me either way, but just double checking.